controllers/portfolio/load_data.py

```
@@ -147,21 +147,33 @@ def _normalize_pairs(df_pos: pd.DataFrame) -> list[tuple[str, str]]:
+        if raw_type in (None, ""):
+            group = "N/D"
+        elif isinstance(raw_type, str):
+            group = raw_type
+        else:
             try:
-                asset_type = psvc.classify_asset_cached(symbol)
+                if pd.isna(raw_type):
+                    group = "N/D"
+                else:
+                    group = str(raw_type)
             except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)